src/component/FetchItems.js

```diff
@@ -8,16 +8,25 @@ import dayjs from 'dayjs';
 
 const FetchItems = ({ token, setToken, firestore }) => {
   const [empty, setEmpty] = useState(true);
+  let itemsDocRef;
+
+  if (!token) {
+    return <Redirect to="" />;
+  } else {
+    itemsDocRef = firestore
+      .collection('lists')
+      .doc(token)
+      .collection('items');
+
+    itemsDocRef.get().then(items => {
+      setEmpty(items.empty);
+    });
+  }
```

Comment:
💡 no need to change, but it's worth nothing that you don't need the `else` because you're already returning inside the `if`.

This would be the equivalent code, but without the else - easier to scan and saves you 1 indentation level!

```js
if (!token) {
  return <Redirect to="" />
}

itemDocRef = firestore....
```

Reply:
Oh that's right! Just a matter of moving it! Thanks @alejandronanez! Looks like Ethan already merged it, but definitely tucking that idea away 👏